libs/go/libp2p_node/utils/utils.go

```diff
@@ -701,7 +701,13 @@ func WriteBytes(s network.Stream, data []byte) error {
 
 	//logger.Debug().Msgf("writing %d", len(data))
 	_, err = wstream.Write(data)
-	wstream.Flush()
+	if err != nil {
+		logger.Error().
+			Str("err", err.Error()).
+			Msg("while sending size")
+		return err
+	}
+	err = wstream.Flush()
```

Comment:
no log in case of `err != nil` after `Flush()`?

Reply:
even not sure logging required here, we pass error and caller can log it if needed